src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>To Do List</title>
+  </head>
+
+  <body>
+
+    <main>
+
+      <div class="status">
+        <div class="error hidden">
+          <span>❌ OOPS!! this task is already added</span>
+        </div>
+        <div class="success hidden">
+          <span>✅ The task is added successfully</span>
+        </div>
+      </div>
```

Comment:
In order to follow the JavaScript best practices (DRY), I would recommend you not to repeat the same div twice, you could leave an empty div and fill it dynamically with JavaScript with the content and style for each case.

Reply:
Yeah! You are right, I will create all this section dynamically using JS.